pyproject.toml

```diff
@@ -1,5 +1,57 @@
 [build-system]
-requires = [
-    "setuptools>=42",
-    "wheel",
-]
\ No newline at end of file
+requires = ["hatchling"]
+build-backend = "hatchling.build"
+
+[project]
+name = "openspoor"
+version = "0.2.9"
```

Comment:
This version can go to something above 0.3.0, maybe 0.3.1 ?  In my previous PR I already upgraded to 0.3.0